### PR TITLE
Swap 2 Previous Owners and add 1 new Owners to `FakeUpgradeSafe`. 

### PR DESCRIPTION
**Pull request by Ethnical**
**Description**

This PR will swap 2 old owners of the FakeUpgradeSafe on Sepolia using the function `swapOwners()` to remove 2 old signers  (See the `README.md`).  And using the function `addOwnerWithThreshold(address owner, uint256 _threshold)` to include 1 new signers.
To do realize this , the task  `021-refresh-signers-FakeFuS` has been created under `sep` folder. 

**Recap**

Using `swapOwners()`: 
This action will perform a swapping of the previous owner that was not active anymore by the new owner.

| PreviousOwner | newOwner |
| --- | --- |
| 0xad70Ad7Ac30Cee75EB9638D377EACD8DfDfE0C3c | 0x95E774787A63f145f7B05028a1479bDc9D055f3d |
| 0xE09d881A1A13C805ED2c6823f0C7E4443A260f2f| 0x41fb1d8c3262e88a056ee3099f5718405CC8cAdE |

Then using addOwnerWithThreshold(address owner, uint256 _threshold) we add 1 owner using function of the safe without changing the threshold.

To Recap the action will be:
| Owner | Threshold |
| --- | --- |
| 0xa03dafade71f1544f4b0120145eec9b89105951f| 2 |

**Tests**
I have run the simulation under tenderly and locally to make sure this is the desired state changes: 
**Tenderly:**
image

**Forge superchain-ops script**
image

**Additional context**
This safes doesn't implement any liveness module, imo this shouldn't trigger any fallback signers. 

**Metadata**
- Fixes https://github.com/ethereum-optimism/safe-wallets/pull/13

### CODE REVIEW

File: tasks/sep/021-refresh-signers-fakeFuS/SignFromJson.s.sol [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        internal
+        view
+        override
+        returns (address[] memory allowed)
+    {
+        allowed = new address[](2);
+        allowed[0] = address(foundationOperationsSafe);
+        allowed[1] = livenessGuard;
+        // allowed[2] = address(0xad70Ad7Ac30Cee75EB9638D377EACD8DfDfE0C3c);
+        // allowed[3] = address(0xE09d881A1A13C805ED2c6823f0C7E4443A260f2f);
```
Comment (mds1) [inline tasks/sep/021-refresh-signers-fakeFuS/SignFromJson.s.sol:64]: delete?
```suggestion
```

File: tasks/sep/021-refresh-signers-fakeFuS/SignFromJson.s.sol [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        Simulation.Payload memory /* simPayload */
+    ) internal view override {
+        console.log("Running post-deploy assertions");
+
+        address[] memory foundationUpgradeSafeOwners = foundationUpgradeSafe
+            .getOwners();
+
+        for (uint256 i = 0; i < foundationUpgradeSafeOwners.length; i++) {
+            require(
+                foundationUpgradeSafeOwners[i] != previousowner1 ||
```
Comment (blmalone) [inline tasks/sep/021-refresh-signers-fakeFuS/SignFromJson.s.sol:82]: I actually think you want this to be an AND condition. With OR, you'll always resort to true. I might've missed the same error in the original PR but I think it was fine regardless. 
```suggestion
                foundationUpgradeSafeOwners[i] != previousowner1 &&
```

File: tasks/sep/021-refresh-signers-fakeFuS/.env [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
```
Comment (blmalone) [inline tasks/sep/021-refresh-signers-fakeFuS/.env:1]: I'm not sure if having multiple directories called `021-*` in `/sep` tasks directory is what we want. Should it increment?

**Review by blmalone — APPROVED** [review]
lgtm, pending one change

File: tasks/sep/021-refresh-signers-fakeFuS/SignFromJson.s.sol (hunk 2 above)
Comment (Ethnical) [reply tasks/sep/021-refresh-signers-fakeFuS/SignFromJson.s.sol]: Oh correct, I have missed it here!

File: tasks/sep/021-refresh-signers-fakeFuS/.env (hunk 3 above)
Comment (Ethnical) [reply tasks/sep/021-refresh-signers-fakeFuS/.env]: Replace the task by `021` -> `023`.

**Review by blmalone — APPROVED** [review]
lgtm
